Prox/Prox/Extensions/UIColorExtensions.swift

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+
+private struct Color {
+    var red: CGFloat
+    var green: CGFloat
+    var blue: CGFloat
+};
+
+extension UIColor {
+    /**
+     * Initializes and returns a color object for the given RGB hex integer.
+     */
```

Comment:
Yeah, this was copy/pasted (and modified) from Firefox iOS. Apologies for the sloppy import!